marm-mcp-server/marm_mcp_server/endpoints/compaction.py

```
@@ -300,6 +301,26 @@ async def _apply_compaction_write(candidate_id: str) -> str:
+                "SELECT suggested_summary FROM compaction_staging WHERE id = ?",
+                (candidate_id,),
+            ).fetchone()
+        if row and row[0]:
+            precomputed_summary = sanitize_content(row[0])
+            precomputed_summary_hash = compute_content_hash(precomputed_summary)
+            if memory._load_encoder_lazily():
+                summary_vec = await asyncio.to_thread(
+                    memory._encode_sync, precomputed_summary
+                )
+                precomputed_summary_embedding = summary_vec.tobytes()
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)